pkg/minishift/openshift/service.go

```diff
@@ -178,6 +212,39 @@ func emptyFilter(data []string) []string {
 	return res
 }
 
+// Get service content in json format
+func getService(namespace string, ip string) (map[string]string, error) {
+	serviceNodePort := make(map[string]string)
+	cmdArgText := fmt.Sprintf("get svc -o json --config=%s -n %s", systemKubeConfigPath, namespace)
+	tokens := strings.Split(cmdArgText, " ")
+	cmdName := instanceState.InstanceConfig.OcPath
+	cmdOut, err := runner.Output(cmdName, tokens...)
+	if err != nil {
+		return nil, err
+	}
+
+	if strings.Contains(byteArrayToString(cmdOut), "No resources found") {
+		return nil, nil
+	}
+
+	var data Service
+	err = json.Unmarshal(cmdOut, &data)
+	if err != nil {
```

Comment:
what are the errors returned here. On a site note, I am concerned about our default mode of handling exceptions. Just checking for it and then passing it on. This might make sense in some cases, but we should also always try to look what type of error can get returned and whether we can do some better error handling.